nbs/tutorials/01_quickdraw.py

```diff
@@ -0,0 +1,36 @@
+from fasthtml.common import *
+from datetime import datetime
+
+db = database('data/drawapp.db')
+rooms = db.t.rooms
+if rooms not in db.t:
+    rooms.create(id=int, name=str, created_at=str, pk='id')
+Room = rooms.dataclass()
+
+@patch
+def __xt__(self:Room):
+    return Li(A(self.name, href=f"/rooms/{self.id}"))
+
+app = FastHTML(hdrs=(picolink,))
```

Comment:
You can replace lines 4-15 with a single call to fast_app, including using `render` (although you'll still need to define that function of course). E.g
```python
app,rt,rooms.Room = FastHTML('data/drawapp.db', render=render, id=int, name=str, created_at=str, pk='id')
```